### ISSUE TITLE
Employee Info (Admin Mode) - Ensure the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds

### ISSUE THREAD

**Issue by will0684** [gcdevops/HRWhiteListing#259]
Title: WCAG 1.4.3: Ensures the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds (span)
Tags: Accessibility, WCAG 1.4.3, color-contrast

Issue: Ensures the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds (color-contrast - https://dequeuniversity.com/rules/axe/3.5/color-contrast?application=msftAI)

Target application: ALEKSANDAR BATINIC - Odoo - http://localhost:8069/web#id=9421&action=137&model=hr.employee&view_type=form&cids=1&menu_id=95

Element path: #o_field_input_480 > span

Snippet: <span>Payment Services Officer</span>

<img width="1679" alt="Screen Shot 2020-04-27 at 10 32 24 AM" src="https://user-images.githubusercontent.com/31868510/80384906-7728ca80-8873-11ea-8f3e-0c141487fde8.png">

How to fix: 
Fix any of the following:
  Element has insufficient color contrast of 3.96 (foreground color: #7c7bad, background color: #ffffff, font size: 11.7pt (15.6px), font weight: normal). Expected contrast ratio of 4.5:1

Environment: Chrome version 80.0.3987.163

====

Title: WCAG 1.4.3: Ensures the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds (span)
Tags: Accessibility, WCAG 1.4.3, color-contrast

Issue: Ensures the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds (color-contrast - https://dequeuniversity.com/rules/axe/3.5/color-contrast?application=msftAI)

Target application: ALEKSANDAR BATINIC - Odoo - http://localhost:8069/web#id=9421&action=137&model=hr.employee&view_type=form&cids=1&menu_id=95

Element path: #o_field_input_481 > span

Snippet: <span>Employment Ins/ops/North York</span>

How to fix: 
Fix any of the following:
  Element has insufficient color contrast of 3.96 (foreground color: #7c7bad, background color: #ffffff, font size: 11.7pt (15.6px), font weight: normal). Expected contrast ratio of 4.5:1

Environment: Chrome version 80.0.3987.163

====

Title: WCAG 1.4.3: Ensures the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds (a[href$="\#notebook_page_443"])
Tags: Accessibility, WCAG 1.4.3, color-contrast

Issue: Ensures the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds (color-contrast - https://dequeuniversity.com/rules/axe/3.5/color-contrast?application=msftAI)

Target application: ALEKSANDAR BATINIC - Odoo - http://localhost:8069/web#id=9421&action=137&model=hr.employee&view_type=form&cids=1&menu_id=95

Element path: a[href$="\#notebook_page_443"]

Snippet: <a data-toggle="tab" disable_anchor="true" href="#notebook_page_443" class="nav-link" role="tab">Work Information</a>

How to fix: 
Fix any of the following:
  Element has insufficient color contrast of 3.96 (foreground color: #7c7bad, background color: #ffffff, font size: 9.8pt (13px), font weight: normal). Expected contrast ratio of 4.5:1

Environment: Chrome version 80.0.3987.163

====

Title: WCAG 1.4.3: Ensures the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds (a[href$="\#notebook_page_456"])
Tags: Accessibility, WCAG 1.4.3, color-contrast

Issue: Ensures the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds (color-contrast - https://dequeuniversity.com/rules/axe/3.5/color-contrast?application=msftAI)

Target application: ALEKSANDAR BATINIC - Odoo - http://localhost:8069/web#id=9421&action=137&model=hr.employee&view_type=form&cids=1&menu_id=95

Element path: a[href$="\#notebook_page_456"]

Snippet: <a data-toggle="tab" disable_anchor="true" href="#notebook_page_456" class="nav-link" role="tab">Access Information</a>

How to fix: 
Fix any of the following:
  Element has insufficient color contrast of 3.96 (foreground color: #7c7bad, background color: #ffffff, font size: 9.8pt (13px), font weight: normal). Expected contrast ratio of 4.5:1

Environment: Chrome version 80.0.3987.163

====

Title: WCAG 1.4.3: Ensures the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds (a[href$="\#notebook_page_465"])
Tags: Accessibility, WCAG 1.4.3, color-contrast

Issue: Ensures the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds (color-contrast - https://dequeuniversity.com/rules/axe/3.5/color-contrast?application=msftAI)

Target application: ALEKSANDAR BATINIC - Odoo - http://localhost:8069/web#id=9421&action=137&model=hr.employee&view_type=form&cids=1&menu_id=95

Element path: a[href$="\#notebook_page_465"]

Snippet: <a data-toggle="tab" disable_anchor="true" href="#notebook_page_465" class="nav-link" role="tab">HR Settings</a>

How to fix: 
Fix any of the following:
  Element has insufficient color contrast of 3.96 (foreground color: #7c7bad, background color: #ffffff, font size: 9.8pt (13px), font weight: normal). Expected contrast ratio of 4.5:1

Environment: Chrome version 80.0.3987.163

====

This accessibility issue was found using Accessibility Insights for Web 2.16.0 (axe-core 3.5.1), a tool that helps find and fix accessibility issues. Get more information & download this tool at http://aka.ms/AccessibilityInsights.
